src/os/ObjectStore.h

```diff
@@ -393,7 +393,10 @@ class ObjectStore {
       __le32 oid;
       __le64 off;
       __le64 len;
-      __le32 dest_cid;
+      union {
+	__le32 fadvise_flags; 		//OP_WRITE
+	__le32 dest_cid;
+      };
       __le32 dest_oid;                  //OP_CLONE, OP_CLONERANGE
```

Comment:
@liewegas What about your idea?

If we determine to refact the Op struct by union, maybe we need a new PR.